packages/medusa/src/api/routes/admin/product-categories/index.ts

```diff
@@ -25,9 +28,25 @@ import updateProductCategory, {
   AdminPostProductCategoriesCategoryParams,
 } from "./update-product-category"
 
+import addProductsBatch, {
+  AdminPostProductCategoriesCategoryProductsBatchReq,
+  AdminPostProductCategoriesCategoryProductsBatchParams,
+} from "./add-products-batch"
+
 const route = Router()
 
 export default (app) => {
+  const atomicTransformQueryConfig = {
```

Comment:
suggestion: Should it be `retrieveTransformQueryConfig` like we have `listTransforn...` wdyt?